integration/ct_integration_test.go

```diff
@@ -43,34 +47,63 @@ func TestCTIntegration(t *testing.T) {
 	fmt.Printf("Today's test has been brought to you by the letters C and T and the number %#x\n", *seed)
 	rand.Seed(*seed)
 
-	cfg, err := ctfe.LogConfigFromFile(*logConfigFlag)
+	cfgs, err := ct.LogConfigFromFile(*logConfigFlag)
 	if err != nil {
 		t.Fatalf("Failed to read log config: %v", err)
 	}
 
+	if err := runParallelTests(cfgs, *httpServersFlag, *testDir, *mmdFlag); err != nil {
```

Comment:
If you'd like to run the same test multiple times under different conditions, please use Go's "Subtests" feature.  Each subtest can be run in parallel to speed things up using `t.Parallel()`
https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks